content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-clipping-example.png" width="800" alt="An example one square kilometer terrain demonstrating clipping due to entity placement." >}}
+
+The following properties are closely related and might require special attention if you have issues with your terrain such as clipping:
+
+* The **Min Height** and **Max Height** properties of the **Terrain World** level component define the vertical range in the level for terrain generation. This range should be large enough to encompass the vertical extents of the box shape of the Terrain Spawner entity. If the box shape extends below the **Min Height** value or above the **Max Height** value, the terrain valleys or peaks are clipped.
+
+* The **Dimensions** property of the **Axis Aligned Box Shape** component specifies the terrain area. The **Dimensions: Z** property, in particular, specifies the height of the terrain. Its value should not exceed the range defined in the **Terrain World** level component by the **Min Height** and **Max Height** properties.
+
+* The **Translate** property of the **Transform** component of the Terrain Spawner entity places the terrain entity in the level. The translate position of the Terrain Spawner entity is at the center of the box shape that defines the terrain area. This means that if the box shape is a 512 unit cube, and the **Translate** property is `0.0` on each axis, then the terrain extends 256 units in each direction from the origin of the level. Care should be taken to ensure that the **Translate: Z** value places the box shape within the range defined by the **Min Height** and **Max Height** properties of the **Terrain World** level component.
+
+## Apply terrain materials
+
+There are two types of materials that are needed for terrain, the *macro material* and *detail materials*.
+
+The macro material is a basic material that applies to the entire terrain. It supports a simple color texture and a normal texture. The macro material is displayed on terrain that is far from the camera.
+
+Detail materials are standard materials that you can create with [Material Editor](/docs/atom-guide/look-dev/materials/material-editor/). They are assigned to the terrain through surface tag names that are generated by gradients, similar to how this tutorial used an image gradient to create elevation. This allows you to assign many materials across a large terrain surface and blend between them. Detail materials are displayed on terrain surfaces that are close to the camera.
+
+{{< tip >}}
+In the **Terrain World Renderer** level component, there is a property group named **Detail material configuration** that you can use to configure how far from the camera to switch from detail material to macro material, and the distance over which to blend between the materials.
```

Comment:
"how far from the camera to stop blending detail materials into the macro material, and the distance over which to fade out the blended detail materials."

Reply:
fixed